feature-libs/cart/quick-order/components/quick-order/quick-order.component.html

```diff
@@ -1,136 +1,165 @@
+              'quickOrderList.warnings.productWasReduced'
+                | cxTranslate
+                  : {
+                      name: warning.entry.product.name,
+                      code: warning.entry.product.code,
+                      quantityAdded: warning.quantityAdded
+                    }
+            }}
+          </span>
+        </li>
+      </ul>
+      <div aria-live="polite" class="cx-visually-hidden">
+        {{
+          'quickOrderList.warnings.reduceWarningFound'
+            | cxTranslate: { count: warnings.length }
+        }}
+      </div>
+    </cx-message>
+  </ng-container>
 
-<ng-container *ngIf="warnings$ | async as warnings">
-  <cx-message
-    *ngIf="warnings.length"
-    (closeMessage)="clearWarnings()"
-    [accordionText]="'quickOrderList.warnings.reviewWarnings' | cxTranslate"
-    [text]="'quickOrderList.warningProceedingToCart' | cxTranslate"
-    [type]="globalMessageType.MSG_TYPE_WARNING"
-    class="quick-order-warnings-message"
-  >
-    <ul class="quick-order-warnings">
-      <li *ngFor="let warning of warnings" class="quick-order-warning-item">
-        <span>
-          {{
-            'quickOrderList.warnings.productWasReduced'
+  <ng-container *ngIf="successes$ | async as successes">
+    <cx-message
+      *ngIf="successes.length"
+      (closeMessage)="clearSuccesses()"
+      [text]="'quickOrderList.successfullyAddedToCart' | cxTranslate"
+      [type]="globalMessageType.MSG_TYPE_CONFIRMATION"
+      class="quick-order-successes-message"
+    >
+      <ul class="quick-order-successes">
+        <li *ngFor="let entry of successes" class="quick-order-success-item">
+          <span>{{
+            'quickOrderList.successes.productAddedToCart'
               | cxTranslate
                 : {
-                    name: warning.entry.product.name,
-                    code: warning.entry.product.code,
-                    quantityAdded: warning.quantityAdded
+                    name: entry.product.name,
+                    code: entry.product.code
                   }
-          }}
-        </span>
-      </li>
-    </ul>
-  </cx-message>
-</ng-container>
-
-<ng-container *ngIf="successes$ | async as successes">
-  <cx-message
-    *ngIf="successes.length"
-    (closeMessage)="clearSuccesses()"
-    [text]="'quickOrderList.successfullyAddedToCart' | cxTranslate"
-    [type]="globalMessageType.MSG_TYPE_CONFIRMATION"
-    class="quick-order-successes-message"
-  >
-    <ul class="quick-order-successes">
-      <li *ngFor="let entry of successes" class="quick-order-success-item">
-        <span>{{
-          'quickOrderList.successes.productAddedToCart'
-            | cxTranslate
-              : {
-                  name: entry.product.name,
-                  code: entry.product.code
-                }
-        }}</span>
-      </li>
-    </ul>
-  </cx-message>
-</ng-container>
+          }}</span>
+        </li>
+      </ul>
+      <div aria-live="polite" class="cx-visually-hidden">
```

Comment:
I think this `aria-live="polite"` attribute is not necessary here, as we have all messages wrapped into another aria-live attribute.